content/manuals/ai/mcp-catalog-and-toolkit/_index.md

```diff
@@ -28,36 +28,59 @@ grid:
 
-- Discovery: A central catalog with verified, versioned tools.
-- Credential management: OAuth-based and secure by default.
-- Execution: Tools run in isolated, containerized environments.
-- Portability: Use MCP tools across Claude, Cursor, Visual Studio Code, and more—no code
-  changes needed.
+Docker MCP Catalog and Toolkit simplifies the developer experience across the following areas:
 
-With Docker Hub and the MCP Toolkit, you can:
+- Discovery: Centralized discovery through a verified catalog of versioned tools.
+- Credential management: Secure credential management with OAuth-based authentication.
+- Execution: Safe execution by running tools in isolated, containerized environments.
+- Portability: True portability. MCP tools work seamlessly across Claude, Cursor, Visual Studio Code, and other platforms without requiring code changes.
 
-- Launch MCP servers in seconds.
-- Add tools using the CLI or GUI.
-- Rely on Docker's pull-based infrastructure for trusted delivery.
+This integrated approach means you can launch MCP servers in seconds, add new tools through either CLI or GUI interfaces, and rely on Docker's pull-based infrastructure for trusted delivery of your tools.
 
-![MCP overview](./images/mcp-overview.svg)
+  ![MCP overview](./images/mcp-overview.svg)
 
-## MCP servers
+To understand how Docker MCP Catalog and Toolkit achieves this streamlined experience, let's explore the underlying architecture and how its key components work together.
 
-MCP servers are systems that use the [Model Context Protocol](https://modelcontextprotocol.io/introduction) (MCP) to help manage
-and run AI or machine learning models more efficiently. MCP allows different
-parts of a system, like the model, data, and runtime environment, to
-communicate in a standardized way. You can see them as
-add-ons that provide specific tools to an LLM.
+## MCP architecture overview
 
-![Example of the GitHub MCP server](./images/mcp-servers-overview.svg)
+MCP follows a client-server architecture that enables standardized communication between AI applications and external tools. This foundational understanding helps you build and integrate MCP solutions effectively.
+
+### MCP servers
+
+MCP servers are specialized programs that provide specific tools and capabilities to AI models through [Model Context Protocol](https://modelcontextprotocol.io/introduction). Each server focuses on a particular domain, like GitHub integration, database access, or file management. You can think of them as extensions that give AI models the ability to interact with external systems and data sources.
+
+### MCP clients
+
+MCP clients are the bridge between AI applications and MCP servers. They enable AI systems to discover, connect to, and interact with external tools and data sources through the standardized MCP protocol. Each client maintains a dedicated one-to-one relationship with a specific MCP server, enabling secure and isolated communication.
```

Comment:
The client-server relationship isn't one-to-one, but many-to-many. One client can be connected to many servers. One server can also be connected to multiple clients.